Refactor "Schedule Friday" for modularity

t-will-gillis opened the pull request:
<!--  Important! Add the number of the issue you worked on  --> 
Fixes #7349 

### What changes did you make?
<!-- Note: add lines if needed, and remove any unused lines -->  
  - Merged the function `minimizeComment()` from `add-label.js` into `github-actions/utils/hide-issue-comment.js` --> both the function and the module are essentially equivalent, edited the formatting to match existing code
  - In `add-label.js`, removed `minimizeComment()` and referenced `hide-issue-comment.js` instead
  - In `schedule-fri-0700.yml`, refactored to remove the double PAT, and removed `pbt` in `main()` in `add-label.js`
  - In `mutate-issue-status.js`, made minor formatting changes by removing unneeded spaces and reformatting error log

### Why did you make the changes (we will use this info to test)?
<!-- Note: add lines if needed, and remove any unused lines -->  
  - Streamlines the code by using module functions and removing extraneous code
  - Although the CodeQL security alert 123 probably is not a concern, this refactor addresses the code 


### Screenshots of Proposed Changes To The Website (if any, please do not include screenshots of code changes)
- No visual changes to website

t-will-gillis committed:
refactor, reuse, and consolidate

t-will-gillis committed:
fix issueNum in query-issue-info.js

t-will-gillis committed:
add HackforLABot login, refs to api, del https ref

t-will-gillis committed:
tweaks to comments

daras-cu reviewed (commented):
Hi @t-will-gillis, I tested this in my repo and got this error when `minimizeIssueComment` was called:
![Sched-Fri-error](https://github.com/user-attachments/assets/3bf0a978-e894-43ad-a5df-7daf9ea84697)

I think it's coming from `hide-issue-comment.js`, the GraphQL mutation is referencing `nodeID` but in the variables it's set as `nodeId`. When I changed the capitalization the action completed and the comments were minimized successfully.

The rest of the changes look good, restructuring/reusing the code from `hide-issue-comment.js` definitely helps streamline `add-label.js` and can be reused again in the future. The only other thing I noticed is you also made some changes to `mutate-issue-status.js` to standardize the formatting, you may want to mention that in the PR.

daras-cu commented on github-actions/utils/hide-issue-comment.js:
```diff
@@ -1,26 +1,29 @@
+  }`;
+
+  const variables = {
+    nodeId: nodeID,
```
This is the `nodeId` referenced in my comment, when changed to `nodeID` the action ran successfully for me.

t-will-gillis replied:
Hey @daras-cu! Thanks for finding the error and pointing this out. I'll resubmit with matching cases. But I will change all to `nodeId` for consistency since elsewhere it is `subjectId` and `clientMutationId`

t-will-gillis committed:
Merge branch 'gh-pages' into refactor-sch-fri-7349

t-will-gillis committed:
Update hide-issue-comment.js

edited `nodeID` --> `nodeId` both to fix an error in mismatched cases and also to make consistent with other `Id`s.

t-will-gillis commented:
Hey @daras-cu Thank you for the review! I addressed the mismatched font casing between `nodeID` and `nodeId`, which hopefully takes care of the error. Also, as you suggested I noted the formatting changes to `mutate-issue-status.js`. Please let me know if you see anything else

daras-cu reviewed (approved):
Looks great @t-will-gillis, the workflow runs with no errors now that the `nodeId` cases match and the formatting all looks consistent to me.
